apps/site/src/views/Schedule/sections/ClipboardSchedule/ClipboardSchedule.module.scss

```diff
@@ -0,0 +1,31 @@
+@use "bootstrap-utils" as bootstrap;
+@use "zothacks-theme" as theme;
+
+.accordion {
+	--bs-accordion-btn-icon: url("~@/assets/icons/accordion-btn.svg");
+	--bs-accordion-btn-active-icon: url("~@/assets/icons/accordion-btn.svg");
+	--bs-accordion-btn-icon-width: 25px;
+}
+
+.clip {
+	position: absolute;
+	width: 75%;
+	top: 0;
+	left: 50%;
+	transform: translate(-50%, -55%);
+
+	@include bootstrap.media-breakpoint-up(lg) {
+		width: 60%;
+	}
+}
+
+.clipboard {
+	background-color: white;
+	border: 1.25rem solid theme.$brown;
+	border-radius: 100px;
```

Comment:
Note: the rounds are looking comically large on narrower views. Consider using a fluid value:
```scss
@include bootstrap.rfs(4rem, border-radius)
```